### PR TITLE
Adding deeplab-mobilenetv2_dm05 models

### PR DESCRIPTION
**Pull request by sayakpaul**
These TF Lite models are by far the fastest ones for doing semantic segmentation but that comes at the cost of sacrificing the quality a bit.

### CODE REVIEW

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_ade20k_train) -->
```
Comment (khanhlvg) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md:5]: isn't it COCO?

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_ade20k_train) -->
```
Comment (khanhlvg) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md:5]: isn't it COCO?

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+<!-- dataset: ADE20k -->
```
Comment (khanhlvg) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md:6]: ditto

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+<!-- dataset: ADE20k -->
```
Comment (khanhlvg) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md:6]: ditto

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md (hunk 1 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md]: Thanks for catching it. Corrected.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md (hunk 2 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md]: Ditto.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md (hunk 3 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md]: Ditto.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md (hunk 4 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md]: Ditto

**Comment by maringeo** [comment]
Hi @sayakpaul, thank you for contributing to TF Hub!

The PR looks good. The validation test failure is in fact due to error in the test itself. I'll push a fix later today. Once this is done I'll rerun the tests and proceed with importing the models in TF Hub.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05/1
```
Comment (maringeo) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md:1]: Looking at the file path structure, it seems like the first line should be
# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1

Note that a placeholder MD files with handle sayakpaul/deeplabv3-mobilenetv2_dm05/1 is defined in tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05/1.md

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05/1
```
Comment (maringeo) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md:1]: Looking at the file path structure, it seems like the first line should be
`# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1`

Note that a placeholder MD files with handle sayakpaul/deeplabv3-mobilenetv2_dm05/1 is defined in tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05/1.md

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md (hunk 5 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md]: Taken care of.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md (hunk 6 above)
Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-int8/1.md]: Taken care of.

File: tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_dm05_coco_voc_trainval) -->
+<!-- dataset: PASCAL VOC 2012 -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
```
Comment (maringeo) [inline tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md:8]: Sorry, this is not documented yet, but the in general the `language` metadata property is usually used for text models to specify the language they were trained or what language their input is expected to be in. 

I think that we can remove the language tag for these set of image models - but please correct me if I'm wrong (I might be missing something regarding how the model is trained or intended to be used).

Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md]: @maringeo there's nothing language-specific in these image models. I just removed it.

**Comment by sayakpaul** [comment]
Hi @maringeo. Just having a follow up on this.

**Comment by maringeo** [comment]
Hi @sayakpaul, the models have been successfully published on tfhub.dev:

https://tfhub.dev/sayakpaul/lite-model/deeplabv3-mobilenetv2_dm05/1/default/1
https://tfhub.dev/sayakpaul/lite-model/deeplabv3-mobilenetv2_dm05-int8/1/default/1
https://tfhub.dev/sayakpaul/lite-model/deeplabv3-mobilenetv2_dm05-float16/1/default/1

The architecture label on the model detail page is empty now and it should show PASCAL VOC 2012 instead - this is a known issue and should be fixed next week.

This PR has been merged in commit 715e5c1e7cd89a862f7259d30dae6bb51b71ac27.

**Comment by sayakpaul** [comment]
Thank you @maringeo for letting me know. Is there a reason as to why this only appears for this architecture but not for the others? 

Cc: @khanhlvg

**Pull request closed without merging**